core/dbt/contracts/graph/nodes.py

```diff
@@ -342,7 +342,6 @@ class ParsedNode(NodeInfoMixin, ParsedNodeMandatory, SerializableType):
     docs: Docs = field(default_factory=Docs)
     patch_path: Optional[str] = None
     build_path: Optional[str] = None
-    deferred: bool = False
```

Comment:
As mentioned, I don't think this is necessary anymore. However:
- This does require updating a lot of tests
- It's exactly the sort of breaking change (removing a field) that we said we would try to avoid doing for artifacts in v1.8+ ...